test/suites/basic.sh

```diff
@@ -697,11 +697,18 @@ EOF
   IPV4_GW="$(lxc network get lxdbr0 ipv4.address | cut -d/ -f1)"
   IPV6_GW="$(lxc network get lxdbr0 ipv6.address | cut -d/ -f1)"
 
+  # Sometimes ping is not installed by cloud-init, so try to install it manually
+  for m in c1 c2 ; do
+    lxc exec micro01 -- lxc exec "${m}" -- apt-get install -y --no-install-recommends iputils-ping
```

Comment:
When launching an `ubuntu-minimal:22.04`, it looks we also have to run `apt-get update` first. I fear cloud-init might have done it so the install works but maybe sometimes it's so slow that it didn't even update the package lists.

As part of https://github.com/canonical/microcloud/pull/436 the images are now pre-fetched on the host (runner) and `security.devlxd.images=true` is set on the `micro*` VMs.
Whilst you are at it can you use the `ubuntu-minimal-daily` instead to fetch the image directly from the host instead of downloading it again? I think the `basic` suite is the only other one that does these workload tests.

Reply:
Fixed.